frappe/social/doctype/energy_point_log/energy_point_log.py

```diff
@@ -187,15 +187,16 @@ def get_alert_dict(doc):
 	return alert_dict
 
 
-def create_energy_points_log(ref_doctype, ref_name, doc, apply_only_once=False):
+def create_energy_points_log(
+	ref_doctype, ref_name, doc, is_energy_point_rule=False, apply_only_once=False
```

Comment:
New args should always be added at end to avoid breaking other code. Force kwargs otherwise.